mne/stats/tests/test_regression.py

```diff
@@ -120,6 +121,8 @@ def test_continuous_regression_no_overlap():
                   events, event_id, tmin, tmax, decim=2)
 
 
+@requires_sklearn
+@testing.requires_testing_data
 def test_continuous_regression_with_overlap():
```

Comment:
add error testing: 
- solver string only in cholesky
- solver output has right shape
- non callable solver throw error